BusBoard/BusBoard/Coordinate.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Newtonsoft.Json;
+
+namespace BusBoard
+{
+    public class Coordinate
+    {
+        [JsonProperty("longitude")] public float Longitude { get; set; }
```

Comment:
If you make the changes to your `Execute` method, you will be able to remove the `JsonProperty` attributes here